azure-mgmt-network/src/main/java/com/azure/management/network/NetworkSecurityGroup.java

```diff
@@ -16,6 +16,7 @@
 import com.azure.management.resources.fluentcore.arm.models.GroupableResource;
 import com.azure.management.resources.fluentcore.arm.models.Resource;
 import com.azure.management.resources.fluentcore.model.Appliable;
+import com.azure.management.resources.fluentcore.model.Attachable;
```

Comment:
Is these `import` necessary, given it is already compile pass?

Reply:
It fixes a warning for https://github.com/azure/azure-libraries-for-java/blob/a3595380aabbcc6e2fa97e6f52480a6d5d0d6393/azure-mgmt-network/src/main/java/com/azure/management/network/NetworkSecurityGroup.java#L85